cogeo_mosaic/scripts/cli.py

```diff
@@ -92,6 +93,68 @@ def create(
         click.echo(json.dumps(mosaicjson.dict(exclude_none=True)))
 
 
+@cogeo_cli.command(short_help="Create mosaic definition from GeoJSON collection")
+@cligj.features_in_arg
+@click.option("--output", "-o", type=click.Path(exists=False), help="Output file name")
+@click.option(
+    "--minzoom",
+    type=int,
+    help="An integer to overwrite the minimum zoom level derived from the COGs.",
+    required=True,
+)
+@click.option(
+    "--maxzoom",
+    type=int,
+    help="An integer to overwrite the maximum zoom level derived from the COGs.",
+    required=True,
+)
+@click.option("--property", type=str, help="Define accessor property", required=True)
+@click.option(
+    "--quadkey-zoom",
+    type=int,
+    help="An integer to overwrite the quadkey zoom level used for keys in the MosaicJSON.",
+)
+@click.option("--min-tile-cover", type=float, help="Minimum % overlap")
+@click.option(
+    "--tile-cover-sort", help="Sort files by covering %", is_flag=True, default=False
+)
+@click.option(
+    "--quiet",
+    "-q",
+    help="Remove progressbar and other non-error output.",
+    is_flag=True,
+    default=False,
+)
+def create_from_features(
+    features,
+    output,
+    minzoom,
+    maxzoom,
+    property,
+    quadkey_zoom,
+    min_tile_cover,
+    tile_cover_sort,
+    quiet,
+):
+    """Create mosaic definition file."""
+    mosaicjson = MosaicJSON.from_features(
+        list(features),
+        minzoom,
+        maxzoom,
```

Comment:
Just pass these as keyword args so that you don't need to add more positional arguments to `from_features`

Reply:
nope because thoses are required in the actual model.

Reply:
Just use this: https://github.com/developmentseed/cogeo-mosaic/pull/63/files#diff-7c1d4d4b7c5cc98334549838d79b4534R109-R136

Don't make them required from the command line, and tell people that if the properties don't exist, an error will be raised.

Reply:
I don't think it's a good way for now. Happy to review this later